frontend/src/pages/ExploreData/ExploreDataPage.tsx

```diff
@@ -317,28 +322,24 @@ function ExploreDataPage(props: ExploreDataPageProps) {
         className={'h-full bg-exploreBgColor'}
       >
         <div
-          className={`z-almostTop mb-1 bg-white p-4 shadow-raised-tighter  transition-all duration-200 ease-in-out md:top-0 md:w-full
-            ${isSticking ? 'md:fixed' : ''}
+          ref={madlibRef}
+          className={`z-almostTop mb-1 bg-white p-4 shadow-raised-tighter md:top-0 md:w-full
+            ${!noTopicChosen ? 'md:sticky' : ''}
           `}
           id='madlib-container'
         >
           <MadLibUI
-            isSticking={isSticking}
             madLib={madLib}
             setMadLibWithParam={setMadLibWithParam}
           />
 
-          {showStickyLifeline && isSticking && (
+          {showStickyLifeline && isStickyEnabled && (
             <p className='flex justify-center '>
               <a href={urlMap.lifeline}>988lifeline.org</a>
             </p>
           )}
         </div>
-        <div
-          className={`w-full pt-0 transition-all duration-200 ${
-            isSticking ? 'md:pt-[150px]' : ''
```

Comment:
This extra top padding with the transition is causing of the "jumping" behavior.